src/components/Clock/Clock.tsx

```diff
@@ -12,51 +16,43 @@ enum ClockHand {
   ss,
 }
 
-const getDegreeByDate = (time: Date, hande: ClockHand): number => {
+const getDegreeByDate = (m: any, hande: ClockHand): number => {
   if (hande === ClockHand.hh) {
-    return time.getHours() * 30;
+    return m.hours() * 30;
   }
   if (hande === ClockHand.mm) {
-    return time.getMinutes() * 6;
+    return m.minute() * 6;
   }
   if (hande === ClockHand.ss) {
-    return time.getSeconds() * 6;
+    return m.second() * 6;
   }
   return 0;
 };
 
 export default function Clock(props: ClockProps) {
-  const [time, setTime] = useState(props.time);
-
-  useEffect(() => {
-    setTimeout(() => {
-      document.title = `You clicked ${time} times`;
-      setTime((t) => {
-        let t2 = t.setSeconds(t.getSeconds() + 1);
-        return new Date(t2);
-      });
-    }, 1000);
-  }, [time]);
+  const [time] = useState(props.time);
+  const [tz] = useState(props.tz);
+  const [difftime] = useState(props.difftime);
 
   return (
     <>
       <div className={styles.clockwrapper}>
         <div className={styles.clock}>
           <ClockFace />
           <div
-            style={{ transform: `rotate(${getDegreeByDate(time, ClockHand.hh)}deg)` }}
+            style={{ transform: `rotate(${getDegreeByDate(moment(time).tz(tz), ClockHand.hh)}deg)` }}
```

Comment:
you can probably create moment object once in a `const` and then re-use in 3 different places here. it'll be a bit better for performance